README.md

```diff
@@ -77,22 +94,114 @@ In the sub-folders for plots that intersected with the orthomosaic, there will b
 The CSV files both contain much of the same information.
```

Comment:
```suggestion
This will generate one directory per plot in the `outputs/` folder.
Each plot will contain two key outputs of interest:
1. `orthomosaic_mask.tif`
2. `canopycover.csv` with the canopy cover calculated from the mask file
   * [In the future](https://github.com/AgPipeline/issues-and-projects/issues/210), these CSV files will be aggregated into a single file for each run.
```

not commenting on `canopycover_geostreams.csv` as this should be deprecated.